libraries/AP_HAL_ChibiOS/hwdef/SkystarsH7HD-bdshot/hwdef.dat

```diff
@@ -18,7 +18,6 @@ define STM32_I2C_USE_DMA FALSE
 
 define RELAY2_PIN_DEFAULT 81 # Pit-1
 define RELAY3_PIN_DEFAULT 82 # PIN-EN
-define RELAY4_PIN_DEFAULT 83 # CAM-C
```

Comment:
oem insists this  is a real camera contrast control,allthough I have not found where its pinned out and oem has not provided the info and details yet
probably premature to remove it yet

Reply:
It is to support this https://github.com/betaflight/betaflight/wiki/FPV-Camera-Control-(Joystick-Emulation) - via the OSD pin. We do not support this.

Reply:
thanks